Push device model updates to udmi_target topic

grafnu opened the pull request.

noursaidi commented:
This doesn't seem to be taking the intended device_id and device_registry_id

```
                      DATA                      │     MESSAGE_ID    │ ORDERING_KEY │             ATTRIBUTES             │ DELIVERY_ATTEMPT │ ACK_STATUS │
├─────────────────────────────────────────────────┼───────────────────┼──────────────┼────────────────────────────────────┼──────────────────┼────────────┤
│ {                                               │ 10321232746707519 │              │ deviceId=ZZ-TRI-FECTA              │                  │ SUCCESS    │
│   "timestamp" : "2020-05-01T13:39:07Z",         │                   │              │ deviceNumId=1689068430253          │                  │            │
│   "version" : "1.4.1",                          │                   │              │ deviceRegistryId=UDMI-REFLECT      │                  │            │
│   "upgraded_from" : "1",                        │                   │              │ deviceRegistryLocation=us-central1 │                  │            │
│   "cloud" : {                                   │                   │              │ projectId=bos-platform-dev         │                  │            │
│     "auth_type" : "ES256",                      │                   │              │ publishTime=2024-02-28T16:54:10Z   │                  │            │
│     "credentials" : [ ]                         │                   │              │ source=clearblade-iot-core         │                  │            │
│   },                                            │                   │              │ subFolder=update                   │                  │            │
│   "system" : {                                  │                   │              │ subType=model                      │                  │            │
│     "location" : {                              │                   │              │ transactionId=PS:10566671557349320 │                  │            │
│       "site" : "ZZ-TRI-FECTA",                  │                   │              │                                    │                  │            │
│       "section" : "2-3N8C"                      │                   │              │                                    │                  │            │
│     },                                          │                   │              │                                    │                  │            │
│     "physical_tag" : {                          │                   │              │                                    │                  │            │
│       "asset" : {                               │                   │              │                                    │                  │            │
│         "guid" : "drw://TBB",                   │                   │              │                                    │                  │            │
│         "site" : "ZZ-TRI-FECTA",                │                   │              │                                    │                  │            │
│         "name" : "GAT-123"                      │                   │              │                                    │                  │            │
│       }                                         │                   │              │                                    │                  │            │
│     },                                          │                   │              │                                    │                  │            │
│     "min_loglevel" : 300                        │                   │              │                                    │                  │            │
│   },                                            │                   │              │                                    │                  │            │
│   "gateway" : {                                 │                   │              │                                    │                  │            │
│     "proxy_ids" : [ "SNS-4", "AHU-22" ]         │                   │              │                                    │                  │            │
│   },                                            │                   │              │                                    │                  │            │
│   "pointset" : {                                │                   │              │                                    │                  │            │
│     "points" : {                                │                   │              │                                    │                  │            │
│       "globulating_globar" : {                  │                   │              │                                    │                  │            │
│         "units" : "Bars"                        │                   │              │                                    │                  │            │
│       }                                         │                   │              │                                    │                  │            │
│     }                                           │                   │              │                                    │                  │            │
│   }                                             │                   │              │                                    │                  │            │
│ }                                               │                   │              │                                    │                  │            │

```

grafnu commented:
Ok, I fixed that problem and likely some other cases where the same thing was happening and we just didn't notice. PTAL! I also updated the logs (see below) to include these parameters so it should be obvious from the logs what is happening too.

```
peringknife@peringknife-glaptop4:~/udmi$ udmis/bin/pod_logs 
Getting logs for pods udmis-pods-79b6f77cdd-2855n udmis-pods-79b6f77cdd-457v4 udmis-pods-79b6f77cdd-xmdkr
Combined logs available in /home/peringknife/udmi/udmis/out/pods.log
peringknife@peringknife-glaptop4:~/udmi$ fgrep "PubSubPipe Published " /home/peringknife/udmi/udmis/out/pods.log | head -n 20
udmis-pods-79b6f77cdd-457v4 2024-02-28T19:37:45Z xxxxxxxx D: PubSubPipe Published PubSub config/udmi to grafnu~udmi_target as null/null null -> PS:10565218916742975
udmis-pods-79b6f77cdd-457v4 2024-02-28T19:37:45Z xxxxxxxx D: PubSubPipe Published PubSub config/udmi to grafnu~udmi_target as null/null null -> PS:10567894606944149
udmis-pods-79b6f77cdd-xmdkr 2024-02-28T19:37:55Z xxxxxxxx D: PubSubPipe Published PubSub config/udmi to grafnu~udmi_target as null/null null -> PS:10565736006482786
udmis-pods-79b6f77cdd-xmdkr 2024-02-28T19:37:56Z xxxxxxxx D: PubSubPipe Published PubSub config/udmi to grafnu~udmi_target as null/null null -> PS:10565685020396698
udmis-pods-79b6f77cdd-2855n 2024-02-28T19:38:05Z xxxxxxxx D: PubSubPipe Published PubSub config/udmi to grafnu~udmi_target as null/null null -> PS:10563899157467325
udmis-pods-79b6f77cdd-2855n 2024-02-28T19:38:06Z xxxxxxxx D: PubSubPipe Published PubSub config/udmi to grafnu~udmi_target as null/null null -> PS:10565178923261664
udmis-pods-79b6f77cdd-2855n 2024-02-28T19:42:56Z 479332bd D: PubSubPipe Published PubSub model/update to grafnu~udmi_target as grafnu~ZZ-TRI-FECTA_9/SNS-4 RC:1709149376618 -> PS:9373431163841264
udmis-pods-79b6f77cdd-2855n 2024-02-28T19:42:56Z cb0fe111 D: PubSubPipe Published PubSub model/update to grafnu~udmi_target as grafnu~ZZ-TRI-FECTA_9/GAT-123 RC:1709149376617 -> PS:10563691230373523
udmis-pods-79b6f77cdd-xmdkr 2024-02-28T19:42:57Z 197e9525 D: PubSubPipe Published PubSub model/update to grafnu~udmi_target as grafnu~ZZ-TRI-FECTA_9/ACME-2301 RC:1709149376621 -> PS:10565973250522469
udmis-pods-79b6f77cdd-xmdkr 2024-02-28T19:42:57Z 203f051a D: PubSubPipe Published PubSub model/update to grafnu~udmi_target as grafnu~ZZ-TRI-FECTA_3/ACME-2301 RC:1709149377204 -> PS:10565795525073650
udmis-pods-79b6f77cdd-457v4 2024-02-28T19:42:57Z 34926528 D: PubSubPipe Published PubSub config/update to grafnu~udmi_target as grafnu~ZZ-TRI-FECTA_9/ACME-2301 RC:1709149377250 -> PS:10563801002647736
udmis-pods-79b6f77cdd-457v4 2024-02-28T19:42:57Z 53bf32c1 D: PubSubPipe Published PubSub config/update to grafnu~udmi_target as grafnu~ZZ-TRI-FECTA_3/AHU-1 RC:1709149377540 -> PS:10567093841769082
udmis-pods-79b6f77cdd-457v4 2024-02-28T19:42:57Z 64964e59 D: PubSubPipe Published PubSub model/update to grafnu~udmi_target as grafnu~ZZ-TRI-FECTA_9/AHU-1 RC:1709149376622 -> PS:10565303423954409
udmis-pods-79b6f77cdd-2855n 2024-02-28T19:42:57Z 6deb44e4 D: PubSubPipe Published PubSub config/update to grafnu~udmi_target as grafnu~ZZ-TRI-FECTA_9/AHU-1 RC:1709149377256 -> PS:10567083263141555
udmis-pods-79b6f77cdd-457v4 2024-02-28T19:42:57Z 739ca453 D: PubSubPipe Published PubSub model/update to grafnu~udmi_target as grafnu~ZZ-TRI-FECTA_3/GAT-123 RC:1709149377203 -> PS:9373711111234297
udmis-pods-79b6f77cdd-457v4 2024-02-28T19:42:57Z 9dedd12e D: PubSubPipe Published PubSub config/update to grafnu~udmi_target as grafnu~ZZ-TRI-FECTA_9/GAT-123 RC:1709149377114 -> PS:10562644606479402
udmis-pods-79b6f77cdd-2855n 2024-02-28T19:42:57Z aa2e643e D: PubSubPipe Published PubSub config/update to grafnu~udmi_target as grafnu~ZZ-TRI-FECTA_3/GAT-123 RC:1709149377650 -> PS:10564029126365719
udmis-pods-79b6f77cdd-2855n 2024-02-28T19:42:57Z b65bfbdd D: PubSubPipe Published PubSub config/update to grafnu~udmi_target as grafnu~ZZ-TRI-FECTA_9/SNS-4 RC:1709149377017 -> PS:10565041124178552
udmis-pods-79b6f77cdd-xmdkr 2024-02-28T19:42:57Z d82fde30 D: PubSubPipe Published PubSub model/update to grafnu~udmi_target as grafnu~ZZ-TRI-FECTA_3/AHU-1 RC:1709149377200 -> PS:10565911737530322
udmis-pods-79b6f77cdd-2855n 2024-02-28T19:42:58Z 19b9cbc8 D: PubSubPipe Published PubSub model/update to grafnu~udmi_target as grafnu~ZZ-TRI-FECTA_3/SNS-4 RC:1709149377205 -> PS:10565441980739345
```
